docs/alexa-directives.rst

```diff
@@ -210,6 +210,129 @@ An APL Command is sent with the `alexaAPLCommand` key in your controller. Just l
 
 
 
+Alexa Presentation Language - T (APLT) Templates
```

Comment:
is this correct? seems like it should be part of a separate PR cause it's not related to the google session entities

Reply:
Removed from this PR.